src/paths.ts

```diff
@@ -64,20 +64,20 @@ class Paths {
     });
   }
 
-  get cmakePath(): Promise<string> { return this._getCMakePath(); }
+  get cmakePath(): Promise<string|null> { return this._getCMakePath(); }
   get ctestPath(): Promise<string> { return this._getCTestPath(); }
 
   private async _getCTestPath(): Promise<string> {
     const ctest_path = config.raw_ctestPath;
     if (!ctest_path || ctest_path == 'auto') {
       const cmake = await this.cmakePath;
-      return path.join(path.dirname(cmake), 'ctest');
+      return path.join(path.dirname(cmake!), 'ctest');
```

Comment:
Don't do a non-null assertion here. Instead, return `null` for the CTest path in the same way as cmakePath.

Reply:
Done